lib/tasks/one_off_tasks/fix_location.rake

```diff
@@ -0,0 +1,31 @@
+namespace :fix_location do
+  desc "fix locations currently in the database"
+
+  task all_previous_records: :environment do
+    jobs_with_locations
+  end
+
+  def jobs_with_locations
+    stackoverflow_jobs = Job.where('url LIKE ?', '%stackoverflow%').all
+    stackoverflow_jobs.each do |job|
```

Comment:
Probably want to batch this or use for_each to go through the jobs, so you don't overtax heroku